app/sprinkles/account/src/Controller/AccountController.php

```diff
@@ -37,6 +38,28 @@
  */
 class AccountController extends SimpleController
 {
+    /**
+     * The route used to check passwords against a list of compromised passwords.
+     *
+     * AuthGuard: false
+     * Route: /account/check-password
+     * Route Name: {none}
+     * Request Type: POST
+     * @see https://haveibeenpwned.com/API/v2
+     * @param Request  $request
+     * @param Response $response
+     * @param array    $args
+     */
+    public function checkPasswordApi(Request $request, Response $response, $args)
```

Comment:
Not too sure about the name of this function. I think `checkPassword` would suffice, and revising the comment to be open to later expansion is worth considering. In a similar vein to comment above re the result type and if a class should be considered, the response shape may also need some revision.

I haven't checked, but its also worth checking that if UF has a standard API response shape that it is being reflected here (I thinking the answer is no currently).

Reply:
> Not too sure about the name of this function. I think checkPassword would suffice,

I am not sure, is it bad practice to have two different functions in different classes with the same name? 

> and revising the comment to be open to later expansion is worth considering.

How about something like: 
``` 
Checks password use eligibility. 
```

Reply:
I updated the name to `checkPassword` and believe I have addressed the other ideas as well.